<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefDialog</class>
 <widget class="QDialog" name="PrefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Charges</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>250</width>
         <height>35</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Order Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="orderNumberLE">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>238</width>
         <height>183</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Open Ends Price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Flanges Charge</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Galvanisisg Minimum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Spray Paint Minimum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Powder Cote Minimum</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QDoubleSpinBox" name="openEndsPriceSB">
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="flangesChargeSB">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="galvanisingMinSB">
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sprayPaintMinSB">
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="powderCoteMinSB">
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>240</width>
         <height>183</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Both Ends Closed Price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>One End Closed Price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Galvanising P/KG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Spray Paint P/SqM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Powder Cote P/SqM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QDoubleSpinBox" name="bothEndsClosedPriceSB">
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>35.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="oneEndClosedPriceSB">
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>30.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="galvanisingPKG_SB">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.350000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sprayPaintPMS_SB">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="powderCotePMS_SB">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QCheckBox" name="openAtStartup">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>20</y>
         <width>141</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Open at Stortup</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Labour and Margins</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>86</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Profit Margin</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="profitMarginSB">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>76</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Material Prices</string>
      </attribute>
      <widget class="QDoubleSpinBox" name="kgFor0_6mmSB">
       <property name="geometry">
        <rect>
         <x>343</x>
         <y>80</y>
         <width>76</width>
         <height>31</height>
        </rect>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <double>99.989999999999995</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>134</y>
         <width>109</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>3.0 mm KG/sqM</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kgFor1_6mmSB">
       <property name="geometry">
        <rect>
         <x>343</x>
         <y>117</y>
         <width>76</width>
         <height>31</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>12.560000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>109</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>0.6 mm KG/sqM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>107</y>
         <width>109</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1.6 mm KG/sqM</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kgFor3_0mmSB">
       <property name="geometry">
        <rect>
         <x>343</x>
         <y>154</y>
         <width>76</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>22.840000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>284</y>
         <width>128</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Price Sheet 3.0mm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>257</y>
         <width>128</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Price Sheet 1.6mm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>230</y>
         <width>128</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Price Sheet 0.6mm</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="priceFor0_6mmSB">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>230</y>
         <width>76</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="priceFor1_6mmSB">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>260</y>
         <width>76</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="priceFor3_0mmSB">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>290</y>
         <width>76</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>orderNumberLE</tabstop>
  <tabstop>bothEndsClosedPriceSB</tabstop>
  <tabstop>oneEndClosedPriceSB</tabstop>
  <tabstop>galvanisingPKG_SB</tabstop>
  <tabstop>sprayPaintPMS_SB</tabstop>
  <tabstop>powderCotePMS_SB</tabstop>
  <tabstop>openEndsPriceSB</tabstop>
  <tabstop>flangesChargeSB</tabstop>
  <tabstop>galvanisingMinSB</tabstop>
  <tabstop>sprayPaintMinSB</tabstop>
  <tabstop>powderCoteMinSB</tabstop>
  <tabstop>kgFor0_6mmSB</tabstop>
  <tabstop>kgFor1_6mmSB</tabstop>
  <tabstop>kgFor3_0mmSB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrefDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrefDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
